src/Revit/RevitHyparTools/Create.Wall.cs

```diff
@@ -29,13 +27,18 @@ public static Elements.Wall[] WallsFromRevitWall(ADSK.Wall wall, Document doc)
             }
 
             var wallPlane = wallFace as PlanarFace;
-            var profiles = GetProfilesOfFace(wallPlane);
+            var profiles = Utils.GetScaledProfilesOfFace(wallPlane);
 
             var centerline = (wall.Location as LocationCurve).Curve;
             var line = new ElemGeom.Line(centerline.GetEndPoint(0).ToVector3(), centerline.GetEndPoint(1).ToVector3());
+            line = Utils.ScaleLineFtToMeters(line);
+            
 
-            var walls = profiles.Select(p => new WallByProfile(line, p, wall.Width));
+            var walls = profiles.Select(p => new WallByProfile(Utils.ReverseProfile(p),
+                                                               wall.Width * Utils.FT_TO_METER_FACTOR,
```

Comment:
Can we use the built in `Elements.Units` class to get these scaling factors? I believe the method would be `FeetToMeters(...)`.

Reply:
yea, didn't know that was there.  done.